src/components/spatial/Spatial.js

```diff
@@ -260,13 +398,18 @@ export default class Spatial extends AbstractSelectableComponent {
     if (images && this.images.length === 0) {
       this.images = Object.entries(images);
     }
-
+    if (tiff && this.tiff.length === 0) {
+      this.tiff = tiff;
+    }
     // Append each layer to the list.
     const layerList = [];
 
     if (images && clearPleaseWait) clearPleaseWait('images');
     layerList.push(...this.renderImageLayers());
 
+    if (tiff && clearPleaseWait) clearPleaseWait('tiff');
+    layerList.push(this.createTIFFLayer());
```

Comment:
Maybe this should also be an array? With images, it was only because the each channel was a separate image, but it feels like a degree of flexibility that might be worth preserving.

Reply:
I don't think we'll ever need more than one of these layers.  If so, the API creating the layer would change to specify that we are.